docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,79 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 2 different identifiers for each participant or subject: 
```

Comment:
should we mention the name `candidate` in this introductory line? (If we go ahead and adopt the term candidate later on. I just remember you mentioning something like that yesterday during the roadmap meeting ;). I have no preference).

Reply:
I'm gonna change it to candidate but just for reference we agreed and subject for the futur